packages/docusaurus/src/client/exports/ComponentCreator.tsx

```diff
@@ -22,7 +23,7 @@ function ComponentCreator(
   if (path === '*') {
     return Loadable({
       loading: Loading,
-      loader: () => import('@theme/NotFound'),
+      loader: () => Promise.resolve(NotFound),
```

Comment:
Somehow if I don't do this then the chunk file will always be missing when doing the server build. Would like some guidance on where are the pieces for code splitting...